.travis.yml

```diff
@@ -1,6 +1,16 @@
 language: rust
-rust: nightly
-script: cd test-project && cargo test
+rust:
+ - stable
+ - nightly
+matrix:
+ include:
+   - rust: stable
+     env: FEATURES="--features stable"
```

Comment:
I would like to add a third build in the matrix to build with `--features stable` on the nightly channel, since that is the most common setup I am using downstream.

Reply:
Done.